python/packages/ai/.pylintrc

```diff
@@ -439,7 +441,9 @@ disable=raw-checker-failed,
         R0903,
         W0212,
         W0511,
-        R1702
+        R1702,
+        W0622,
```

Comment:
This rule was for "too many branches" which doesn't seem super valuable to me right now since we are porting code that already exists. Later on if we want to remove these and refactor some things we can.